web/services/hooks/organizations.tsx

```diff
@@ -1,21 +1,30 @@
 import { useSupabaseClient, useUser } from "@supabase/auth-helpers-react";
 import { useQuery } from "@tanstack/react-query";
-import { Members } from "../../pages/api/organization/[id]/members";
-import { Owner } from "../../pages/api/organization/[id]/owner";
 import { Database } from "../../supabase/database.types";
 import { useCallback, useEffect, useState } from "react";
 import Cookies from "js-cookie";
 import { OrgContextValue } from "../../components/layout/organizationContext";
 import { ORG_ID_COOKIE_KEY } from "../../lib/constants";
+import { useJawnClient } from "../../lib/clients/jawnHook";
 
 const useGetOrgMembers = (orgId: string) => {
+  const jawn = useJawnClient();
```

Comment:
Let's replace this with 
`getJawnClient(orgId)`

So we have less hooks on the context